src/main/java/uk/gov/hmcts/reform/sscs/helper/ResourceLoader.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.hmcts.reform.sscs.helper;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+import java.net.URL;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public final class ResourceLoader {
```

Comment:
I can see resource loader being used in the tests, so followed similar approach. 
I would want to put this in properties file rather than a json and pick it up, but was advised to keep it in json.
Is it worth keeping it in application.properties and the injecting them using @{Value?

Reply:
Yes you can read directly into the object mapper using Paths, File, or URL

Reply:
Removed ResourceLoader, instead reading the json via - 

Resource resource = new ClassPathResource("multipleHearingLocations.json");

I would ideally want to  inject it using @value

@Value("classpath:multipleHearingLocations.json")
private Resource resource;

But at the moment I am struggling to inject this in tests. If this works, we would be able to get rid of the checked IOexception as well.

Reply:
You can try constructor injection instead

Reply:
Constructor injection requires a whole number of changes in several locations in this case. The HearingDetailsMapping class cannot be instantiated and employs static methods. It's not a spring bean. I'm generally against this pattern among other things exactly because of the ones we're experiencing here. Unfortunately reviewing the architectural decisions that led to create this class as a collection of static methods is outside the scope of this ticket.

I'm going to make a call here and say that we can disregard this specific issue. Instead, these changes you're suggesting we will put up as technical debt. The loading of the resource direcly through `new ClassPathResource` is good enough for now.